File: DevExpress-Demo/DevExpress.SalesDemo.Wpf/CS/DevExpress.SalesDemo.Wpf/ViewModel/RegionsViewModel.cs
using DevExpress.Mvvm.POCO;

namespace DevExpress.SalesDemo.Wpf.ViewModel {
    public class RegionsViewModel : PageViewModel {
        public static RegionsViewModel Create() {
            return ViewModelSource.Create(() => new RegionsViewModel());
        }
        protected RegionsViewModel() { }

        public virtual PerformanceBarChartViewModel DailySalesByRegionViewModel { get; protected set; }
        public virtual PerformanceBarChartViewModel UnitSalesByRegionViewModel { get; protected set; }
        public virtual PieBarRangeViewModel PieGaugeRangeViewModel { get; protected set; }
        protected override void Initialize() {
            DailySalesByRegionViewModel = PerformanceBarChartViewModel.Create(Modules.Regions, PerformanceViewMode.Daily);
            UnitSalesByRegionViewModel = PerformanceBarChartViewModel.Create(Modules.Regions, PerformanceViewMode.Monthly);
            PieGaugeRangeViewModel = PieBarRangeViewModel.Create(Modules.Regions);
        }
    }
}

File: DevExpress-Demo/DevExpress.SalesDemo.Wpf/VB/DevExpress.SalesDemo.Wpf/ViewModel/RegionsViewModel.vb
Imports DevExpress.Mvvm.POCO

Namespace DevExpress.SalesDemo.Wpf.ViewModel

    Public Class RegionsViewModel
        Inherits PageViewModel

        Public Shared Function Create() As RegionsViewModel
            Return ViewModelSource.Create(Function() New RegionsViewModel())
        End Function

        Protected Sub New()
        End Sub

        Public Overridable Property DailySalesByRegionViewModel As PerformanceBarChartViewModel

        Public Overridable Property UnitSalesByRegionViewModel As PerformanceBarChartViewModel

        Public Overridable Property PieGaugeRangeViewModel As PieBarRangeViewModel

        Protected Overrides Sub Initialize()
            DailySalesByRegionViewModel = PerformanceBarChartViewModel.Create(Modules.Regions, PerformanceViewMode.Daily)
            UnitSalesByRegionViewModel = PerformanceBarChartViewModel.Create(Modules.Regions, PerformanceViewMode.Monthly)
            PieGaugeRangeViewModel = PieBarRangeViewModel.Create(Modules.Regions)
        End Sub
    End Class
End Namespace
